README.md

```diff
@@ -62,13 +62,25 @@ For more configuration options see [here](../../wiki/Configuration).
 
 Need more details? You can find more detailed installation guides for [Unix](../../wiki/Unix) (including your Raspberry Pi), [macOS](../../wiki/macOS), and [Windows](../../wiki/Windows).
 
-#### Building via Docker
-
-*Note*: Using this method, the bot will lose all sounds when restarting because the Docker container has its own file system.
-
-+ Simply clone the repo and run `docker build -t soundbot .` inside the folder.
-+ Afterwards start the bot via `docker run soundbot`.
-+ To run the container in the background use `docker run -d soundbot`.
+#### Building/Running via Docker
+
++ First, clone the repo and run `docker-compose build` inside the folder.
++ Make a separate directory where your soundbot's data will live (Ex. `/etc/discord-soundbot`)
++ Then create a `docker-compose.yml` file in that directory with the following contents:
+```yaml
+version: '2'                                           
+services:                                              
+  discord-soundbot:                                    
+    image: soundbot
```

Comment:
This image should be updated to `markokajzer/discord-soundbot` whenever that image is set up.  The step about building above could also be removed at that point, since they could just download the pre-built image.